app/src/main/java/com/x8bit/bitwarden/data/auth/repository/AuthRepositoryImpl.kt

```diff
@@ -1377,17 +1373,15 @@ class AuthRepositoryImpl(
             if (isDeviceUnlockAvailable) {
                 unlockVaultWithTdeOnLoginSuccess(
```

Comment:
Can we also update the docs for all these existing helpers for now to make clear they are unlocking the vault for the active user? So changing

```kotlin
    /**
     * Attempt to unlock the vault with trusted device specific data.
     */
    private suspend fun unlockVaultWithTdeOnLoginSuccess(
        loginResponse: GetTokenResponseJson.Success,
        userStateJson: UserStateJson,
        deviceData: DeviceDataModel?,
    ): VaultUnlockResult? {
```

to something like

```kotlin
    /**
     * Attempt to unlock the vault for the current user with trusted device specific data.
     */
```

or

```kotlin
    /**
     * Attempt to unlock the current user's vault with trusted device specific data.
     */
```